ansible_deploy/command_line.py

```diff
@@ -263,15 +279,51 @@ def setup_ansible(setup_hooks, commit):
         else:
             logger.error("Not supported")
 
-def run_task(config, options):
+
+def run_task(options: dict, inventory: str):
     """
     Function implementing actual execution of ansible-playbook
     """
+    command = ["ansible-playbook", "-l", options["infra"], "-i", inventory, options["task"]]
```

Comment:
Need loop over play_items with lookup to play_items list for actual playbook and eventual check if stage is not on skipped list